src/lib/configure.js

```diff
@@ -57,3 +67,11 @@ function getDefaultApiAddr ({ protocol, host, port }) {
 
   return `${protocol || 'http'}://${host || 'localhost'}:${port || 5001}`
 }
+
+// returns the passed function wrapped in a function that ignores
+// undefined values in the passed `options` object
+function wrap (fn, defaults) {
+  return (input, options) => {
+    return fn(input, mergeOptions(defaults, options))
+  }
+}
```

Comment:
Suggestion:
```js
const wrap = (fn, defaults) => (input, options) => fn(input, mergeOptions(defaults, options));
```

Reply:
I find this sort of deeply chained single-line-arrow-function style optimises readability for computers and not humans.  It'll get minified to something similar to this anyway so there's little to gain here.